benchmarks/bayesmark/report_bayesmark.py

```diff
@@ -1,6 +1,6 @@
 import abc
 from collections import defaultdict
-import io
+from dataclasses import dataclass
```

Comment:
I have a question. Optuna still supports Python 3.6. Should we also support Python 3.6 for benchmark scripts? Or has it already been dropped?

Reply:
I think we don't have to support Python 3.6 in the benchmark. In https://github.com/optuna/optuna/issues/3021, we have decided not to support Python 3.6 in integration modules and examples.

Reply:
Thank you for letting me know!